elastic/kibana
# [Response Ops][Task Manager] Deprecating `xpack.task_manager.claim_strategy` config

Resolves https://github.com/elastic/kibana/issues/202492

## Summary

For 9.0 only. Deprecates `xpack.task_manager.claim_strategy` configuration. Logs warning if value is explicitly set. This value was never added to the Kibana docs in `docs/settings/task-manager-settings.asciidoc` so nothing to remove there.

## To Verify

Set `xpack.task_manager.claim_strategy` config in `kibana.dev.yml` and start Kibana. See warning get logged.
